oauth2client/contrib/_metadata.py

```diff
@@ -29,8 +29,13 @@
 from oauth2client import transport
```

Comment:
Please bump up the minor version here https://github.com/gsutil-mirrors/oauth2client/blob/ce6bd3d80624ef22f558fe8518f0d592dbe71863/oauth2client/__init__.py#L17